conf/messages.en

```diff
@@ -194,3 +194,9 @@ reportAlternativeCharge.checkYourAnswersLabel = Alternative charge applied
 reportAlternativeCharge.checkYourAnswersLabel.change=Change alternative charge applied
 reportAlternativeCharge.change.hidden = Alternative charge applied
 
+
+searchEngineLoss.title = Has the group made a loss on its search engine activity?
```

Comment:
@tapiwa-tiyemba group is a placeholder, please refer to https://github.com/hmrc/digital-services-tax-frontend/blob/59aa5b47c4dff15677d717754fa1d26c50a164ca/conf/messages#L638
This is the page that we are replicating in returns frontend